.github/workflows/ci.yml

```diff
@@ -46,9 +46,9 @@ jobs:
           - name: Linux py311
             os: ubuntu-latest
             pyversion: '3.11'
-          - name: Linux py312
-            os: ubuntu-latest
-            pyversion: '3.12'
+          # - name: Linux py312  Not yet, because zmq
```

Comment:
strange, `pyzmq` released wheels for 3.12: https://pypi.org/project/pyzmq/#files

https://github.com/zeromq/pyzmq/issues/1891

Our `fastplotlib` notebook tests are passing on 3.12, I'll double check locally with a new env with just jupyter-rfb

Reply:
Hmm. This is the CI log for the failed build: https://github.com/vispy/jupyter_rfb/actions/runs/6628132857/job/18004557295

Reply:
Pypi lists wheels for 3.12. Why does it try to build the wheel for zmq then?

Reply:
Ah, because it uses pyzmq 24.0.1, which does not have these wheels.

Reply:
I don't have the time to look into this now. Will merge as is to fix CI. Can look into py312 another time

Reply:
ah strange, I'll look into it later

Reply:
The logs show that it first selects and downloads 25.1.1, but a little later it picks 24.0.1. Strange indeed.

Reply:
I wonder if it's the jupyterlab pin, I'll test this in a separate PR. jupyterlab v4 has been working for us for months. I can do this in a separate PR.

https://github.com/vispy/jupyter_rfb/blob/main/pyproject.toml